Does not build on 32bit OS [6270]

Just tried compiling FastRTPS (HEAD at fa77f1319e7b37869b66a29e47ea1cf4916260fe) from source on Raspberry Pi 4 (Raspbian Release 10, Buster) and it fails on 

```
[  1%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/persistence/SQLite3PersistenceService.cpp.o
/home/pi/projects/third_party/Fast-RTPS/src/cpp/rtps/persistence/SQLite3PersistenceService.cpp: In member function ‘virtual bool eprosima::fastrtps::rtps::SQLite3PersistenceService::add_writer_change_to_storage(const string&, const eprosima::fastrtps::rtps::CacheChange_t&)’:
/home/pi/projects/third_party/Fast-RTPS/src/cpp/rtps/persistence/SQLite3PersistenceService.cpp:182:78: error: ‘const struct eprosima::fastrtps::rtps::SequenceNumber_t’ has no member named ‘to64long’
  182 |         sqlite3_bind_int64(add_writer_change_stmt_, 2, change.sequenceNumber.to64long());
      |                                                                              ^~~~~~~~
/home/pi/projects/third_party/Fast-RTPS/src/cpp/rtps/persistence/SQLite3PersistenceService.cpp: In member function ‘virtual bool eprosima::fastrtps::rtps::SQLite3PersistenceService::remove_writer_change_from_storage(const string&, const eprosima::fastrtps::rtps::CacheChange_t&)’:
/home/pi/projects/third_party/Fast-RTPS/src/cpp/rtps/persistence/SQLite3PersistenceService.cpp:213:81: error: ‘const struct eprosima::fastrtps::rtps::SequenceNumber_t’ has no member named ‘to64long’
  213 |         sqlite3_bind_int64(remove_writer_change_stmt_, 2, change.sequenceNumber.to64long());
      |                                                                                 ^~~~~~~~
/home/pi/projects/third_party/Fast-RTPS/src/cpp/rtps/persistence/SQLite3PersistenceService.cpp: In member function ‘virtual bool eprosima::fastrtps::rtps::SQLite3PersistenceService::update_writer_seq_on_storage(const string&, const eprosima::fastrtps::rtps::GUID_t&, const eprosima::fastrtps::rtps::SequenceNumber_t&)’:
/home/pi/projects/third_party/Fast-RTPS/src/cpp/rtps/persistence/SQLite3PersistenceService.cpp:270:63: error: ‘const struct eprosima::fastrtps::rtps::SequenceNumber_t’ has no member named ‘to64long’
  270 |         sqlite3_bind_int64(update_reader_stmt_, 4, seq_number.to64long());
      |                                                               ^~~~~~~~
make[2]: *** [src/cpp/CMakeFiles/fastrtps.dir/build.make:1740: src/cpp/CMakeFiles/fastrtps.dir/rtps/persistence/SQLite3PersistenceService.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:987: src/cpp/CMakeFiles/fastrtps.dir/all] Error 2
make: *** [Makefile:163: all] Error 2

```

--- a/include/fastrtps/rtps/common/SequenceNumber.h
+++ b/include/fastrtps/rtps/common/SequenceNumber.h
@@ -63,8 +63,6 @@ struct RTPS_DllAPI SequenceNumber_t
     {
     }
 
-    // Check the target support 64bits.
-#ifdef LLONG_MAX
     /*! Convert the number to 64 bit.
      * @return 64 bit representation of the SequenceNumber
      */
@@ -72,7 +70,6 @@ struct RTPS_DllAPI SequenceNumber_t
     {
         return (((uint64_t)high) << 32) + low;
     }
-#endif
 
     /*!
      * Assignment operator
@@ -343,10 +340,10 @@ inline bool sort_seqNum(const SequenceNumber_t& s1, const SequenceNumber_t& s2) 
 }
 
 /**
- * 
+ *
  * @param output
  * @param seqNum
- * @return 
+ * @return
  */
 inline std::ostream& operator<<(std::ostream& output, const SequenceNumber_t& seqNum)
 {

